src/main/java/org/opentripplanner/routing/impl/GraphPathFinder.java

```diff
@@ -358,6 +351,7 @@ private RoutingRequest createReversedRequest(RoutingRequest originalReq, Routing
         reversedOptions.maxTransfers = 4;
         reversedOptions.longDistance = true;
         reversedOptions.bannedTrips = options.bannedTrips;
+        reversedOptions.bannedTripSequences = options.bannedTripSequences;
```

Comment:
Just spitballing here, but do the sequences need to be reversed? If not, a comment might be nice.

Reply:
Yes. Reversed requests happen in most every request to try to find optimal journeys.